Makefile

```
@@ -5,14 +5,14 @@ setup_dev_environment:
 	poetry run pre-commit install
 
 test:
```

Can we not just simplify this command to have pytest gather tests dynamically?
I've tried it and it successfully gathered all the tests and executed them in about 42s compared to 57s

```suggestion
test:
	poetry run python -m pytest de_workloads
	poetry run python -m pytest datastacks
```